packages/currency/src/currency_types/IotTokens.ts

```diff
@@ -0,0 +1,9 @@
+import BaseCurrencyType from './BaseCurrencyType'
+
+const TICKER = 'IOT'
```

Comment:
```suggestion
export const TICKER = 'IOT'
```

Reply:
They are exposed via the Currency library. You can access them using `CurrencyType.mobileToken.ticker` etc or from the Balance class itself